src/components/task-step/ends.cjsx

```diff
@@ -69,6 +69,15 @@ HomeworkEnd = React.createClass
   onNextStep: ->
     @setState({})
 
+  renderCompletionMessage: (completeSteps, totalSteps) ->
+    congratsMessage = <h1>You are done! Great job!</h1> if completeSteps is totalSteps
+
+    [
```

Comment:
Any reason this is not a `<div>`? That way it is possible to style easier later.